pallets/pallet-bonded-coins/README.md

```diff
@@ -1,4 +1,137 @@
+
+### Feature guided 
+
+- `BenchmarkHelper`: Helper type for benchmarking to calculate asset ids for collateral and bonded currencies
+
+## Life Cycle of a Pool
+
+1. **Creation**:
+   - A pool is created using the `create_pool` function.
+   - The manager specifies the bonding curve, collateral type, currencies, and whether the bonded coins are transferable.
+   - A deposit is taken from the caller, which can be reclaimed once the pool is destroyed.
+
+2. **Refund Process**:
+   - The refund process can be started by the manager using the `start_refund` function.
+   - The refund process can be forced to start using the `force_start_refund` function, which requires force privileges.
+   - Collateral can be refunded to a specific account using the `refund_account` function, based on the owned bonded currency.
+
+3. **Destruction**:
+   - The destruction process can be started by the manager using the `start_destroy` function. This operation will fail if accounts with bonded currencies still exist.
+   - The destruction process can be forced to start using the `force_start_destroy` function, which requires force privileges. This operation will fail if there are still accounts with bonded currencies.
+   - The destruction process is completed using the `finish_destroy` function, which refunds any taken deposits.
+
+## Functions
+
+### Public Functions
+
+- `create_pool`: Creates a new pool with the specified bonding curve, collateral type, and currencies. During the pool creation step, the manager must specify whether the bonded coins are transferable or not. This flag can not be changed later on. The selected denomination is used not only as metadata but also to scale down the existing supply or the amount specified in a mint or burn operation. An deposit is taken from the caller, which can be reclaimed back, once the pool is getting destroyed. 
+- `reset_team`: Resets the managing team of a pool. Only the admin and the freezer can be updated in this operation. 
+- `reset_manager`: Resets the manager of a pool. 
+- `set_lock`: Sets a lock on a pool. Locks specify who is able to mint and burn a bonded currency. After applying the lock, the pool becomes permissioned, and only the manager is able to mint or burn bonded currencies.
+- `unlock`: Unlocks the pool. 
+- `mint_into`: Mints new tokens by locking up collateral. In the mint_into operation the beneficiary must be specified. The collateral is taken from the caller. 
+- `burn_into`: Burns tokens to release collateral. In the burn_into operation the beneficiary must be specified. The funds are burned from the caller.
+- `start_refund`: Starts the refund process for a pool. Only the manager is able to start the refund process. 
+- `force_start_refund`: Forces the start of the refund process for a pool. Requires force privileges. 
+- `refund_account`: Refunds collateral to a specific account. The amount of refunded collateral is determined by the owned bonded currency
```

Comment:
```suggestion
- `refund_account`: Refunds collateral to a specific account. The amount of refunded collateral is determined by the owned bonded currency.
```

Reply:
Updated in: [ea7228e](https://github.com/KILTprotocol/kilt-node/pull/807/commits/ea7228e002fa608ba2742d4b7442c92f73994433)